src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1411,7 +1411,7 @@ def _download(targets: List[str]) -> None:
         return os.path.join(path, jobname + "0." + preference)
 
     @protect_grpc
-    def _ctrl(self, cmd):
+    def _ctrl(self, cmd: str, opt1: str = ""):
         """Issue control command to the mapdl server
 
         Available commands:
```

Comment:
You really need to use double backticks so that the commands are formatted as code entities in the doc (rather than shown in italics).